File: C++/Product.cpp
#include <string>
#include <iostream>

using namespace std;

class Product
{
    private:
        string idProduct;
        string price;
    public:
        Product()
        {

        }

        Product(string idProduct, string price)
        {
            this->idProduct = idProduct;
            this->price = price;
        }

        void setIdProduct(string idProduct)
        {
            this->idProduct = idProduct;
        }

        string getIdProduct()
        {
            return this->idProduct;
        }

        void setPrice(string price)
        {
            this->price = price;
        }

        string getPrice()
        {
            return this->price;
        }

        ~Product()
        {

        }
};

File: PHP/Product.php
<?php

class Product
{
    private $idProduct;
    private $price;

    function __construct($idProduct = null, $price = null)
    {
        $this->idProduct = $idProduct;
        $this->price = $price;
    }

    function setIdProduct($idProduct)
    {
        $this->idProduct = $idProduct;
    }

    function getIdProduct()
    {
        return $this->idProduct;
    }

    function setPrice($price)
    {
        $this->price = $price;
    }

    function getPrice()
    {
        return $this->price;
    }
}

?>